Cairo Desktop/Cairo Desktop/Taskbar.xaml.cs

```diff
@@ -175,55 +175,64 @@ public void setPosition()
             double screenHeight = Screen.Bounds.Height / dpiScale;
 
             setTopPosition(Screen.Bounds.Bottom / dpiScale);
-            
-            this.Left = Screen.Bounds.Left / dpiScale;
 
-            this.bdrTaskbar.MaxWidth = screenWidth - 36;
-            this.Width = screenWidth;
+            Left = Screen.Bounds.Left / dpiScale;
+
+            if (Settings.FullWidthTaskBar)
```

Comment:
Shouldn't this use the full width of the screen instead, and maybe apply some style differences to account for that? This looks a bit awkward and has undesirable results when the taskbar fills up currently.

Reply:
This should be better now

Reply:
It is!